build/controllers/github.js

```diff
@@ -150,6 +150,8 @@ module.exports = {
       switch (request.payload.action) {
         case 'opened':
           return pullRequestOpenedWebhook(request);
+        case 'reopened':
+          return pullRequestOpenedWebhook(request);
```

Comment:
```suggestion
        case 'reopened':
        case 'opened':
          return pullRequestOpenedWebhook(request);
```
Comme pour l'instant on fait la même chose en cas de réouverture que d'ouverture, je propose de simplifier le switch case en visualisant programmatiquement que l'action reopened déclenche le même traitement que l'action opened